components/lsp/createLSPContractInstance.ts

```diff
@@ -4,7 +4,7 @@ import { ethers } from "ethers";
 export default function createLSPContractInstance(
   signer: ethers.Signer,
   contractAddress: string
-) {
+): ethers.Contract {
```

Comment:
This one is very annoying. We should disable the `Missing return type on function` rule across all our repos 😄

Reply:
Implicit returns are fine most of the time, I'd agree. I don't really like the defaults of the next linter.